distributed/client.py

```diff
@@ -3579,6 +3579,12 @@ def get_worker_logs(self, n=None, workers=None, nanny=False):
         """
         return self.sync(self.scheduler.worker_logs, n=n, workers=workers, nanny=nanny)
 
+    def log_event(self, topic, msg):
+        return self.sync(self.scheduler.log_event, topic=topic, msg=msg)
```

Comment:
Open question, what should the name of this be?  `log_event` is perfectly fine.  Do we want something that claims more space, like `log`?

Reply:
I have a slight preference for `log_event` as (to me) it conveys there's some additional structure beyond, for example, just logging a string in Python's normal logging system. Additionally `log` and `get_logs` are already in use elsewhere. We could certainly move things around, but I'm just not sure if it'd be worth the effort to do it right now